xbmc/guilib/GUIControlFactory.cpp

```diff
@@ -1174,6 +1174,10 @@ CGUIControl* CGUIControlFactory::Create(int parentID, const CRect &rect, TiXmlEl
         labelInfo, scrollOut, timeToPauseAtEnd, resetOnLabelChange);
 
       ((CGUIFadeLabelControl *)control)->SetInfo(infoLabels);
+
+      // check whether or not a scroll tag was specified.
+      if (scrollValue != CGUIControl::FOCUS)
+        ((CGUIFadeLabelControl *)control)->SetScrolling(scrollValue == CGUIControl::ALWAYS ? true : false);
```

Comment:
No need for the ?: part. `scrollValue == CGUIControl::ALWAYS` evaluates to exactly that